android/build.gradle

```diff
@@ -190,12 +177,10 @@ dependencies {
   implementation project(":react-native-reanimated")
 }
 
-if (isNewArchitectureEnabled()) {
-  react {
-    jsRootDir = file("../src/")
-    libraryName = "LiveMarkdown"
-    codegenJavaPackageName = "com.expensify.livemarkdown"
-  }
+react {
+  jsRootDir = file("../src/")
+  libraryName = "LiveMarkdown"
+  codegenJavaPackageName = "com.expensify.livemarkdown"
 }
```

Comment:
This shouldn't be neccessary, due to `codegenConfig` field in `package.json`

Reply:
What exactly do you suggest? Should I remove only `codegenJavaPackageName` or something more?

Reply:
I believe entire `react` block isn't needed

Reply:
Cool, I'll remove that, we still need `apply plugin: "com.facebook.react"` though, right?

Reply:
Removed in 8b3d88b.

Reply:
Removed in 8b3d88b.

Reply:
Can you check if updating `codegenConfig` to
```
  "codegenConfig": {
    "name": "RNLiveMarkdownSpec",
    "type": "all",
    "jsSrcsDir": "src",
    "android": {
      "javaPackageName": "com.expensify.livemarkdown"
    }
  }
```
helps?

Reply:
That helped. Changed in 1f60bf2.